src/main/java/org/cyclonedx/gradle/utils/ExternalReferencesUtil.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * This file is part of CycloneDX Gradle Plugin.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) OWASP Foundation. All Rights Reserved.
+ */
+package org.cyclonedx.gradle.utils;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
```

Comment:
In addition to my suggestion below, two imports would need to be added.

```suggestion
import java.io.IOException;
import java.io.InputStream;
import java.io.InputStreamReader;
```

Reply:
This is likely overkill for this shell command. However, in other cases you can run into issues if one of the streams (error or output) fills up it can cause the process to block/hang.